src/lib/modules/server/terminal/holder-client.ts

```
@@ -89,40 +94,44 @@ export class HolderClient {
 							scrollback = msg.data;
+						} else {
+							// Queue activity/cwd/exit messages received during handshake
+							pendingMessages.push(msg);
 						}
 
 						// Handshake complete once we have info.
 						if (info !== null) {
-							if (msg.type === 'scrollback' || info.exited) {
-								// Got scrollback or PTY already exited — resolve immediately.
+							const settle = () => {
+								if (settled) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)